CHANGELOG.md

```diff
@@ -6,6 +6,80 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.9.0] Io - 2022-02-22 [:warning:]
+Injectable wire encoding and a `protobuf` wire encoder.
+
+### Added
+
+- :sparkles: **Injectable wire encoding [#233]:** The encoding used for messages that are sent across the network is now injectable.
+  - Add binary marshalling: [#272], [#284], [#298].
+  - Add serializer interface: [#297], [#325].
+  - Changed generation of `PropoposalID`: [#300], [#307].
+  - Rename message types [#305].
+  - Extend and refactor message serialization tests: [#316], [#317].
+- :sparkles: **Protobuf wire encoder [#311]:** A wire encoder based on `protobuf` has been added [#318].
+- Minor additions: Introduce `Asset.Equal` [#279], export proposer index [#313].
```

Comment:
This is the only entry missing an emoji. On purpose?

Reply:
Yes, and on purpose. Thought a minor addition was not worth a :sparkles:, they would be reserved for the major ones. ;)